lib/utils.pm

```diff
@@ -427,6 +427,8 @@ anymore for storage-ng.
 
 =cut
 sub workaround_type_encrypted_passphrase {
+    # for Leap 42.3 the boot partition is not encrypted, therefore password is not asked
+    return if is_leap('=42.3') && get_var('UEFI') && !is_storage_ng && get_var('ENCRYPT') && get_var('LVM');
```

Comment:
Seems that  `=42.3` does not work, made a change and waiting for: http://openqa-staging-1.qa/tests/9 to finish

Reply:
it should work fine.. https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/t/01_version_utils.t#L55

Reply:
@kravciak just figured that I had my workers misconfigured :), and they were not loading the tests from my openQA instance but from a local Path, I'm such a genius:1st_place_medal: